deepspeed/launcher/launch.py

```diff
@@ -190,13 +246,33 @@ def main():
                 raise ValueError("Don't use both the '--no_python' flag"
                                  " and the '--module' flag at the same time.")
         cmd.append(args.training_script)
-        # A user may not want to pass local_rank as a keyword arg so we make this optional.
-        if not args.no_local_rank:
-            cmd.append(f"--local_rank={local_rank}")
         cmd += args.training_script_args
-
-        process = subprocess.Popen(cmd, env=current_env)
-        processes.append(process)
+        cmd_args = cmd[1:]
+
+        rdzv_configs: Dict[str, str] = {'timeout': 100}
+        rdzv_parameters = RendezvousParameters(backend='c10d',
+                                               endpoint=args.master_addr + ":" +
+                                               str(args.master_port),
+                                               run_id='123456789',
```

Comment:
Discussed offline. Let's fetch `run_id` from an env variable, if it's not set revert back to the default you defined in constants `ELASTIC_TRAINING_ID_DEFAULT`.

Reply:
Thanks for catching it.